Jenkinsfile

```diff
@@ -151,6 +151,9 @@ stage('Build') {
 			}
 		})
 	}
+	executions.put('Hibernate Search Update Dependency', {
+		build job: '/hibernate-search-dependency-update/6.2', parameters: [string(name: 'UPDATE_JOB', value: 'orm6.2'), string(name: 'ORM_REPOSITORY', value: helper.scmSource.remoteUrl), string(name: 'ORM_BRANCH', value: helper.scmSource.branch.name)]
```

Comment:
I think you need an extra parameter to actually wait for the build to finish and have the step forward its result (success/failure)?
I don't have the docs readily available, but I remember we mentioned this in the original discussion on Zulip.

Reply:
Also, `helper.scmSource.remoteUrl` will likely be an SSH URL which from what I remember won't work

Reply:
mm Ok, let me figure that out (where we can get a repo URL in the format that would work, and then I'll update the PR)
And thanks, I received the invite 😃

Reply:
> mm Ok, let me figure that out (where we can get a repo URL in the format that would work, and then I'll update the PR)

Alternatively we could make SSH work in the Hibernate Search job. Do you remember what wasn't working exactly?

Reply:
> Do you remember what wasn't working exactly?

hmm I think it was something related to  "unknown host" or something like that... we can trigger the build with a ssh URL and it'll tell us 😃 

The other thing ... I've created a test job here: https://ci.hibernate.org/blue/organizations/jenkins/hibernate-orm-personal-marko/detail/ci%2Fadd-hibernate-search-tests-to-build-test/2/pipeline/23 
And since the Search update job is throttled ... we might be waiting for a while for this build to finish 🥲

Reply:
>  And since the Search update job is throttled ... we might be waiting for a while for this build to finish 🥲

There are ways to bypass throttling. Currently we ignore throttling when the build is triggered by a user. Maybe similar exceptions are possible for builds triggered by other builds?

Reply:
yeah, I was trying to find something like it, but from https://www.jenkins.io/doc/pipeline/steps/workflow-multibranch/ looks like only user boost is available?

Reply:
> > And since the Search update job is throttled ... we might be waiting for a while for this build to finish 🥲
> 
> There are ways to bypass throttling. Currently we ignore throttling when the build is triggered by a user. Maybe similar exceptions are possible for builds triggered by other builds?

Apparently no, there are no such exceptions :/

There are other plugins to handle throttling though, maybe they are more flexible? https://plugins.jenkins.io/throttle-concurrents/

Reply:
I've realized that I didn't post the comment yesterday 🤦🏻‍♂️  I've looked at the other plugin, but it doesn't look like it can do what we need...  
Which leads to a question: do we really need the throttling on that job? (since it already has corn trigger only)

Reply:
>  Which leads to a question: do we really need the throttling on that job? (since it already has corn trigger only)

Corn trigger? :thinking: 

This job is triggered every time we push to the relevant branch, but throttled to once every two weeks.

We can change this to not trigger it on push at all, and have a cron to run it every two weeks. It just means older branches will run for no reason, but I guess that's actually good since we want to test against e.g. newer versions of ORM even if there was no change in Hibernate Search.

Reply:
🤦🏻‍♂️ series continues 🤣 
```
triggers {
		// Run at least once per week, in case of snapshot updates.
		cron '@weekly'
	}
```
cron I meant cron 😄, but then I wouldn't mind to get some 🌽 or 🍿 😆

Reply:
Ok... I think there's also a push trigger, but it's implicit. At least that's how I understand questions like this: https://stackoverflow.com/questions/42798006/how-to-disable-automatic-build-from-scm-change-in-jenkinsfile

Anyway... +1 for doing crons only, removing the rate limitter, and suppressing the scm change trigger. Can you take care of it, please?